src/test/java/com/seals/camplanner/location/controllers/LocationControllerTest.java

```diff
@@ -32,7 +35,7 @@
 @SpringBootTest
 @AutoConfigureMockMvc
 @Transactional
-public class LocationControllerTest {
```

Comment:
As far as I have seen it is fairly standard for test classes and methods to be `public`. Do you have a reasoning for this change?